GOVERNANCE.md

```diff
@@ -0,0 +1,221 @@
+maintainer sponsoring you must have had close work interactions with you and be
+knowledgeable of some of your work. 
+
+To apply, open a PR to update the `OWNERS` file and mention your sponsor in the
+description. The PR description should also list the PRs you have reviewed.
+
+### Responsibilities
+
+- Expected to be responsive to review requests.
+- Stay up to date with the project's direction and goals,
+  e.g., by attending some of the bi-weekly meetings, standups,
+  or being around in the cert-manager-dev Slack channel.
+
+### Privileges
+
+- Can `/approve` on pull requests.
+
+## Maintainer
+
+A maintainer is someone who can communicate with the CNCF on behalf of the
+project and who can participate in lazy consensus and votes. 
+
+**Defined by:** [`MAINTAINERS.md`](./MAINTAINERS.md).
+
+### Becoming a Maintainer
+
+Anyone can become a cert-manager maintainer. Maintainers should be proficient in
+Go; have expertise in at least one of the domains (Kubernetes, PKI, ACME); have
+the time and ability to meet the maintainer expectations above; and demonstrate
+the ability to work with the existing maintainers and project processes.
+
+To become a maintainer, start by expressing interest to existing maintainers.
+Existing maintainers will then ask you to demonstrate the qualifications above
+by contributing PRs, doing code reviews, and other such tasks under their
+guidance. After several months of working together, maintainers will decide
+whether to grant maintainer status.
+
+### Privileges
+
+- Can communicate with the CNCF on behalf of the project.
+- Can participate in lazy consensus and votes.
+
+### Responsibilities
+
+- Monitor cncf-cert-manager-\* emails and help out when possible.
+- Respond to time-sensitive security release processes.
+- Create and attend meetings with the cert-manager Steering Committee (not less than once a quarter).
+- Attend "maintainers vote" meetings when one is scheduled.
+
+### Maintainer Decision-Making
+
+Substantial changes to the project require a "maintainers decision". This includes,
+but is not limited to, changes to the project's roadmap, changes to the project's
+scope, fundamental design decisions, and changes to the project's governance.
+
+A "maintainers decision" is made using lazy consensus. Email or Slack
+can be used to reach lazy consensus as long as the deliberation date
+and time are specified and the maintainers are CC'ed. You may use the
+following message template:
+
+> Dear maintainers, I'd like us to reach an agreement on the following matter using lazy consensus: [...]
+> - 🧑‍💻 Participants: @cert-manager-maintainers
+> - 📢 Deadline: April 3rd, 2023 23:59 UTC  
+> - 🚨 Note: to speed up the process, you may answer with a :+1: or a comment stating that your are lazy to help reach consensus before the deadline.
+
+Any non-agreements with regards to the decision can be posted as comments on the Slack message or to the email thread.
+
```

Comment:
Agreed. I propose:

> While most decisions are typically reached through the principle of lazy consensus, there exists the option for a maintainer to propose a formal vote. Unless otherwise specified, such a vote would require a simple majority approval from all maintainers to be considered successful. Situations that might warrant a formal vote include, but are not limited to, cases where a decision necessitates explicit input from every participant or when disagreements arise during a lazy consensus discussion.

WDYT

Reply:
> I am unsure how to include the "binding" part. Is "binding vote" defined as "a vote that is associated with a justification"?

I think that's my bad, I didn't really mean "binding". I meant a "voting process with mandatory responses". I like what you wrote.

The only other issue I can see with this process is that there's currently no process defined for someone that refuses or consistently forgets to vote despite it being required.

Maybe after some period of time without voting (maybe a week?) the non-voter automatically becomes an "abstain". I think there should also be some kind of repercussion (e.g. if a user doesn't bother to vote for several votes in a row they automatically become an emeritus maintainer). Maybe we don't need to define that yet though!

Reply:
Ah, I get it. I have added the paragraph about the voting procedure, and I haven't specified how we should behave in case someone won't cast their vote quickly enough. I agree, we can figure it out when that happens.